web/react/components/edit_channel_header_modal.jsx

```diff
@@ -27,27 +31,39 @@ export default class EditChannelHeaderModal extends React.Component {
         }
     }
 
+    componentWillReceiveProps(nextProps) {
+        if (this.state.originalHeader !== nextProps.channel.header) {
```

Comment:
That's a good point. I completely forgot that the original header would still be available there. I'll make that change